src/contacto.html

```diff
@@ -1,13 +1,13 @@
 <!DOCTYPE html>
```

Comment:
Why did you remove the indentation inside the `html` tag?

Reply:
Camilo, thanks for your check, the errors that you report me was already fixed.